### PR TITLE
Added index file to export modules folder

### PR DESCRIPTION
**Pull request by marcelorl**
fix for exporting modules folder you had mentioned before.

### CODE REVIEW

File: src/Modules/index.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export {default as pornhub} from './Pornhub';
+export {default as redtube} from './Redtube';
+export {default as sex} from './Sex';
+export {default as xvideos} from './Xvideos';
```
Comment (LucasLeandro1204) [inline src/Modules/index.js]: Please add a newline at end and give spaces between brackets eg 
`export { ... } from ...`

File: src/Pornsearch.js [hunk 2]
```diff
@@ -1,10 +1,7 @@
 import Axios from 'axios';
 import Cheerio from 'cheerio';
 
-import pornhub from './Modules/Pornhub';
-import redtube from './Modules/Redtube';
-import sex from './Modules/Sex';
-import xvideos from './Modules/Xvideos';
+import {pornhub, redtube, sex, xvideos} from './Modules';
```
Comment (LucasLeandro1204) [inline src/Pornsearch.js]: You can import everything as Module, drop Pornsearch.Load and define this.modules = Modules, I think it's better.

Comment (LucasLeandro1204) [reply src/Pornsearch.js]: import Modules from './Modules';

**Comment by marcelorl** [comment]
Updated files.

Took the liberty to install `semistandard` lint package.
You are 100% free to disagree with it and I can revert it then.

**Comment by LucasLeandro1204** [comment]
Yeah, this lint is ok to me xd

**Pull request merged**